### PR TITLE
Mikhalchenko  homework3 task2. Humans.

### PR DESCRIPTION
**Pull request by tatianamih**
Hello colleagues. Could please review the  my changes I made for Hw2 task1.

### CODE REVIEW

**Comment by LytkinaOlga** [comment]
lgfm

File: homeWork_03/ex2_Human/tanyamikhalchenko/src/main/java/jrr/by/mikhalcheno/Human.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+    public Human() {
+    }
+    public Human(String name, int age) {
+        this.name = name;
+        this.age = age;
+    }
+
+    public void  greet() {
+        System.out.println("Hi! My name is " + this.name + ", I'm " + this.age + " years old.");
```
Comment (AnastasiaKichaeva) [inline homeWork_03/ex2_Human/tanyamikhalchenko/src/main/java/jrr/by/mikhalcheno/Human.java:15]: Не this.name, this.age, а name и age

**Review by AnastasiaKichaeva — APPROVED** [review]

**Review by spolwer — APPROVED** [review]
lgfm

**Review by VladimirPetranovski — APPROVED** [review]
lgfm

**Comment by tatianamih** [comment]
My third homework is the second task. I create humans.

**Pull request merged**